src/skai/model/train.py

```diff
@@ -44,6 +45,12 @@
 flags.DEFINE_bool(
     'is_vertex', False, 'True if the training job will be executed on VertexAI.'
 )
+flags.DEFINE_enum(
+  'accelerator_type',
```

Comment:
There should be 2 more spaces here.

Reply:
I have increased the spacing by 2. Thank you